response_operations_ui/views/reporting_units.py

```diff
@@ -29,6 +29,10 @@ def view_reporting_unit(ru_ref):
             collection_exercise['responseStatus'] = map_ce_response_status(collection_exercise['responseStatus'])
             collection_exercise['companyRegion'] = map_region(collection_exercise['companyRegion'])
 
+        for respondent in survey['respondents']:
+            respondent['status'] = map_respondent_status(respondent['status'])
+            respondent['enrolmentStatus'] = map_respondent_status(respondent['enrolmentStatus'])
```

Comment:
Do we need to map both statuses? What's the difference?

Reply:
And if they're different surely they should have separate mapping functions

Reply:
One is the account status and the other is the enrolment status. Could separate it out into two functions but i didn't see any point in doing it.

Reply:
CREATED, ACTIVE, SUSPENDED  are for account
PENDING,  ENABLED, DISABLED,  SUSPENDED are for enrolment